src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp

```diff
@@ -0,0 +1,936 @@
+                    _events.ScheduleEvent(EVENT_SCENE_TALK_COMETH + 27, 500ms);
+                    break;
+
+                case EVENT_SCENE_TALK_COMETH + 27:
+                    _summons.DespawnAll();
+                    Reset();
+                    break;
+
+                default:
+                    break;
+            }
+        }
+
+        if (!UpdateVictim())
+            return;
+
+        DoMeleeAttackIfReady();
+    }
+
+    void SummonPortalsFromOrgrimmar()
+    {
+        for (Position const& pos : OrgrimmarPortalPos)
+            me->SummonCreature(NPC_PORTAL_FROM_ORGRIMMAR, pos, TEMPSUMMON_TIMED_DESPAWN, 300s);
+
+        std::list<Creature*> orgrimmarPortals;
+        GetCreatureListWithEntryInGrid(orgrimmarPortals, me, NPC_PORTAL_FROM_ORGRIMMAR, 100.0f);
+
+        for (Creature* portal : orgrimmarPortals)
+            portal->CastSpell(portal, SPELL_AIR_REVENANT_ENTRANCE);
+    }
+
+    void SummonGarroshAndHisEliteGuards()
+    {
+        for (Position const& pos : HellscreamElitePos)
+            me->SummonCreature(NPC_HELLSCREAM_ELITE_COMETH, pos, TEMPSUMMON_TIMED_DESPAWN, 300s);
+
+        me->SummonCreature(NPC_GARROSH_HELLSCREAM, GarroshPos, TEMPSUMMON_TIMED_DESPAWN, 300s);
+
+        if (Creature* cromush = me->SummonCreature(NPC_HIGH_WARLORD_CROMUSH_COMETH, CromushPos, TEMPSUMMON_TIMED_DESPAWN, 300s))
+        {
+            _cromushGUID = cromush->GetGUID();
+
+            cromush->RemoveNpcFlag(UNIT_NPC_FLAG_QUESTGIVER);
+            cromush->GetMotionMaster()->MovePath(PATH_CROMUSH, false);
+        }
+    }
+
+    void DespawnGarroshAndHisEliteGuards()
+    {
+        for (ObjectGuid const& summonedCreature : _summons)
+        {
+            if (Creature* summon = ObjectAccessor::GetCreature(*me, summonedCreature))
+                summon->CastSpell(summon, SPELL_SIMPLE_TELEPORT);
+        }
+    }
+
+private:
+    EventMap _events;
+    ObjectGuid _garroshGUID;
+    ObjectGuid _cromushGUID;
+    ObjectGuid _sylvanasGUID;
+    ObjectGuid _agathaGUID;
+    SummonList _summons;
+    bool _eventInProgress;
+};
+
+enum RaiseForsakenCometh
+{
+    ACTION_RISE_DURING_RAISE                = 1,
+    ACTION_DESCEND_AFTER_RAISE              = 2,
+
+    POINT_BEING_RISEN                       = 1,
+
+    ANIMKIT_RESET                           = 0,
+    ANIMKIT_FALLEN_HUMAN                    = 721
+};
+
+// Raise Forsaken - 83173
+class spell_silverpine_raise_forsaken_83173 : public AuraScript
+{
+    PrepareAuraScript(spell_silverpine_raise_forsaken_83173);
+
+    void OnApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+    {
+        Unit* target = GetTarget();
+
+        if (Creature* fallenHuman = target->ToCreature())
+        {
+            if (fallenHuman->IsAIEnabled())
+                fallenHuman->AI()->DoAction(ACTION_RISE_DURING_RAISE);
+        }
+    }
+
+    void OnRemove(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+    {
+        Unit* target = GetTarget();
```

Comment:
Target may be null here, you hooked it AfterRemove (may be null), not OnRemove (not null)

Reply:
I just noticed this, and it should be OnRemove instead, just updated it.